RetsTalk/RetsTalk/User/Model/UserSettingManager.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  UserSettingManager.swift
+//  RetsTalk
+//
+//  Created by HanSeung on 11/25/24.
+//
+
+import Combine
+
+final class UserSettingManager: UserSettingManageable, @unchecked Sendable {
+    private var userData: UserData {
+        didSet { userDataSubject.send(userData) }
+    }
+    private(set) var userDataSubject: CurrentValueSubject<UserData, Never>
+    private let userDataStorage: Persistable
+    
+    // MARK: Init method
+
+    init(userData: UserData, persistent: Persistable) {
+        self.userData = userData
+        self.userDataSubject = CurrentValueSubject(userData)
+        self.userDataStorage = persistent
```

Comment:
userDataStorage는 self를 빼도 동작하겠네요!
```suggestion
        userDataStorage = persistent
```